ui/src/views/Settings.vue

```diff
@@ -97,6 +98,7 @@
               :disabled="loadingState"
               :invalid-message="error.nethvoice_admin_password"
               ref="nethvoice_admin_password"
+              :type="passwordFieldType"
```

Comment:
```suggestion
              type="password"
```
`passwordFieldType` has always value `password`, no need to use a variable